doc/_templates/autoapi/python/data.rst

```diff
@@ -34,11 +37,18 @@
     {% endif %}
 
 
+   {%+ if obj.docstring != "autoapi_noindex" -%}
```

Comment:
I think all of these need to be updated to use `in` right?

Reply:
I retained this as `!=` so that if the doctoring is purely `autoapi_noindex`, nothing is rendered. 

However, as you mentioned earlier, going forward, we might have some useful information in these docstrings, in addition to the `autoapi_noindex` tag. 

If we change this to `in`, even the useful information will be missed.